src/solver/optimisation/constraints/Group1.cpp

```diff
@@ -0,0 +1,86 @@
+#include "Group1.h"
+AreaBalanceData Group1::GetAreaBalanceData()
```

Comment:
All these **Get...()** should be removed from class **Group1** (see **Group1.h** 's review).